src/libs/ajax/Apps.js

```diff
@@ -76,5 +76,11 @@ export const Apps = signal => ({
     const res = fetchLeo(`api/apps/v2/${workspaceId}/${appName}`,
       _.mergeAll([authOpts(), jsonBody(body), { signal, method: 'POST' }]))
     return res
+  },
+  deleteAllAppsV2: async (workspaceId, deleteDisk = true) => {
+    const res = await fetchLeo(`api/apps/v2/${workspaceId}/deleteAll${qs.stringify({ deleteDisk }, { addQueryPrefix: true })}`,
```

Comment:
For my own edification, why do we need `addQueryPrefix`?

Reply:
This is needed to prefix the `?`. This file is somewhat inconsistent in it's usage vs. manually including a `?` in the string to be interpolated.